src/utils/generateProjectPermalink.js

```diff
@@ -10,7 +10,7 @@ module.exports = (data) => {
 	}
 
 	const locale = data.locale;
-	const localeSlug = config.languages[locale].slug || locale;
+	const localeSlug = config.languages[locale]?.slug || locale;
```

Comment:
@greatislander the issue Jon was having was caused by this line, because config.languages[locale] was undefined. This was the fix I suggested, but if you have different opinion, please let me know.

Reply:
That looks like a great approach, thanks @chosww and @jhung!